src/lib.rs

```diff
@@ -15,49 +15,53 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![deny(missing_docs)]
 
-//! Efficient and compact serialization of Rust types.
+//! Efficient and space-efficient serialization of Rust types.
 //!
 //! This library provides structures to easily retrieve compile-time type
-//! information at runtime and also to serialize this information in a compact
-//! form.
+//! information at runtime and also to serialize this information in a
+//! space-efficient form, aka `PortableForm`.
 //!
 //! # Registry
 //!
-//! At the heart of its functionality is the [`Registry`](`crate::Registry`) that acts as cache for
-//! known types in order to efficiently deduplicate them and thus compactify the overall
-//! serialization.
+//! At the heart of its functionality is the [`Registry`](`crate::Registry`)
+//! that acts as a cache for known types in order to efficiently deduplicate
+//! types and ensure a space-efficient serialization.
 //!
 //! # Type Information
 //!
-//! Information about types is provided via the [`TypeInfo`](`crate::TypeInfo`) trait.
+//! Information about types is provided via the [`TypeInfo`](`crate::TypeInfo`)
+//! trait.
 //!
 //! This trait should be implemented for all types that are serializable.
-//! For this the library provides implementations for all commonly used Rust
-//! standard types and provides derive macros for simpler implementation of user
-//! provided custom types.
+//! `scale-info` provides implementations for all commonly used Rust standard
+//! types and a derive macro for implementing of custom types.
 //!
-//! # Compaction Forms
+//! # Forms
 //!
-//! There is an uncompact form, called [`MetaForm`](`crate::form::MetaForm`) that acts as a bridge
-//! from compile-time type information at runtime in order to easily retrieve all information needed
-//! to uniquely identify types.
+//! There is an expanded form, called [`MetaForm`](`crate::form::MetaForm`) that
```

Comment:
I don't think `expanded` is the right word here. That said, I'm not sure what the right word would be here either. Could maybe just remove `There is an expanded form` and just use the rest of the sentence. Or something to describe that this is how types are described and identified statically.

Reply:
The whole paragraph is a bit weird imo. Hard to rewrite because I'm not sure I understand it myself. Changing to:

```
//! # Forms
//!
//! To bridge between compile-time type information and runtime the
//! [`MetaForm`](`crate::form::MetaForm`) is used to easily retrieve all
//! information needed to uniquely identify types.
```

Reply:
Feel free to suggest moar betterments, happy to make the changes.